ament_cmake_uncrustify/cmake/ament_uncrustify.cmake

```diff
@@ -11,12 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
Please revert this unrelated whitespace change. The empty line intentionally separates the copyright header from the function docblock.

Reply:
The whitespace change has been reverted.